app/livechat/lib/Assets.js

```diff
@@ -1 +1,2 @@
 export const addServerUrlToIndex = (file) => file.replace('<body>', `<body><script> SERVER_URL = '${ __meteor_runtime_config__.ROOT_URL }'; </script>`);
+export const addCobrowseLicenseKey = (file, key) => file.replace('#LICENSE_KEY', key);
```

Comment:
Two things here:
- LICENSE_KEY is too generic, replace by a most specific term like `SCREEN_SHARE_LICENSE_KEY` or something like that.
- You're mentioning a specific provider function in the core, which isn't the expected way to implement that, you need to call a method from the `ScreenSharing Manager` which will return the information you need, you can't call a method here that is related to a specific provider, that's the reason we have the  `ScreenSharing Manager`.